lib/gitgitgadget.ts

```diff
@@ -164,41 +169,29 @@ export class GitGitGadget {
         return true;
     }
 
-    // better parms would be to pass prinfo object
-    public async submit(pr: IPullRequestInfo, userInfo: IGitHubUser):
+    // Send emails only to the user
+    public async preview(pr: IPullRequestInfo, userInfo: IGitHubUser):
         Promise<string | undefined> {
 
-        // if known, add submitter to email chain
-        const ccSubmitter = userInfo.email ? `\nCc: ${
-            userInfo.name} <${userInfo.email}>` : "";
-        const description = `${pr.title}\n\n${pr.body}${ccSubmitter}`;
-
-        if (pr.baseOwner !== "gitgitgadget" || pr.baseRepo !== "git") {
-            throw new Error(`Unsupported repository: ${pr.pullRequestURL}`);
-        }
+        const send = async (mail: string): Promise<string> => {
+            const mbox = await parseMBox(mail);
+            mbox.cc = [];
+            mbox.to = userInfo.email;
+            return await sendMail(mbox, this.smtpOptions);
+        };
 
-        // get metadata in work repo
-        const metadata =
-            await this.notes.get<IPatchSeriesMetadata>(pr.pullRequestURL);
-        const previousTag = metadata && metadata.latestTag ?
-            `refs/tags/${metadata.latestTag}` : undefined;
-        // update work repo from base
-        await this.updateNotesAndPullRef(pr.number, previousTag);
+        return await this.genAndSend( pr, userInfo, {noUpdate: true}, send);
+    }
 
-        const series =
-            await PatchSeries.getFromNotes(this.notes, pr.pullRequestURL,
-                                           description, pr.baseLabel,
-                                           pr.baseCommit, pr.headLabel,
-                                           pr.headCommit, userInfo.name);
+    // Send emails out for review
+    public async submit(pr: IPullRequestInfo, userInfo: IGitHubUser):
```

Comment:
I feel that it would make the patch more obvious if the refactoring of `submit()`'s code was done in a preparatory commit, and the `preview()` function was introduced only in a subsequent commit. It is a bit hard for me to be certain that I did not miss anything when reviewing such a large patch...